kv/memdb/memdb.go

```diff
@@ -475,6 +508,7 @@ func (ms *mergeState) calculateRecomputeHeight(key []byte, sb *Sandbox) int {
 }
 
 type nodeHeader struct {
+	flags  KeyFlags
 	height uint16
```

Comment:
If we can reduce height to uint8, it'll be more memory friendly.

Reply:
https://github.com/pingcap/tidb/blob/761a961ba0fb2ff8d48deb8b46e417d3d4cbd18f/kv/sandbox.go#L503-L508

Reply:
Afraid of the overflow risk.

Reply:
Never overflow. The upper bound is https://github.com/pingcap/tidb/blob/761a961ba0fb2ff8d48deb8b46e417d3d4cbd18f/kv/sandbox.go#L25